src/main/java/eggventory/Eggventory.java

```diff
@@ -35,10 +37,17 @@ public static void main(String[] args) {
         String currentDir = System.getProperty("user.dir");
         String stockFilePath = currentDir + "/data/saved_stocks.csv";
         String stockTypesFilePath = currentDir + "/data/saved_stocktypes.csv";
+        String loanListFilePath = currentDir + "/data/saved_loanlist.csv";
+        String personListFilePath = currentDir + "/data/saved_personlist.csv";
+        String templateListFilePath = currentDir + "/data/saved_templatelist.csv";
 
-        storage = new Storage(stockFilePath, stockTypesFilePath);
+        storage = new Storage(stockFilePath, stockTypesFilePath, loanListFilePath, personListFilePath,
+                templateListFilePath);
         parser = new Parser();
         stockList = storage.load();
+        loanList = storage.loadLoanList();
+        personList = storage.loadPersonList();
+        templateList = storage.loadTemplateList();
 
```

Comment:
your commits aren't stacking properly... aren't these from the last PR that i just approved?